airflow/example_dags/tutorial_functional_etl_dag.py

```diff
@@ -0,0 +1,111 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=missing-function-docstring
+"""
+### Functional DAG Tutorial Documentation
+
+This is a simple ETL data pipeline example which demonstrates the use of Functional DAGs
+using three simple tasks for Extract, Transform, and Load.
+
+Documentation that goes along with the Airflow Functional DAG tutorial located
+[here](https://airflow.apache.org/tutorial_functional.html)
+"""
+# [START tutorial]
+# [START import_module]
+import json
+
+# The DAG object; we'll need this to instantiate a DAG
+from airflow import DAG
+from airflow.utils.dates import days_ago
+
+# [END import_module]
+
+# [START default_args]
+# These args will get passed on to each operator
+# You can override them on a per-task basis during operator initialization
+default_args = {
+    'owner': 'airflow',
+}
+# [END default_args]
+
+# [START instantiate_dag]
+with DAG(
+    'tutorial_functional_etl_dag',
+    default_args=default_args,
+    description='Functional ETL DAG tutorial',
+    schedule_interval=None,
+    start_date=days_ago(2),
+    tags=['example'],
+) as dag:
+    # [END instantiate_dag]
+
+    # [START documentation]
+    dag.doc_md = __doc__
+    # [END documentation]
+
+    # [START extract]
+    @dag.task()
```

Comment:
You can do 
```suggestion
    @dag.task
```
which is cleaner imo

Reply:
I think this creates mypy issues :<